src/static/css/cards.css

```diff
@@ -0,0 +1,37 @@
+.card-image{
+  display:block;
+  max-width:35% !important;
```

Comment:
Sometimes wrapping them in an element with an id then targeting them as children of that elem can work ...